src/styles/blocks/catalog.scss

```diff
@@ -18,6 +17,14 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform color 300ms ease-out;
```

Comment:
transition is not applied for cards, also consider using variable for animation time :)

Reply:
I have made a variable for transition duration, thanks for the tip :) . However, I do not understand the first part of your remark. In my code, transition is applied to every card with hover status (although it is added under the general selector, as per point 1 of the checklist). What should I change?

Reply:
it is not about proper place but way of providing particular properties. (https://prnt.sc/TRQSyyF-2B0e) Try changing it to "all" and see how it works :)

Reply:
Now I get it, I corrected the code accordingly :)